kairon/shared/chat/user_media.py

```
@@ -542,24 +547,12 @@ def get_media_ids(bot: str):
-            obj = UserMediaData.objects.get(
+            UserMediaData.objects(
                 bot=bot,
                 media_id=media_id
-            )
-            filename = obj.output_filename
-            if not bucket:
-                bucket = Utility.environment["storage"]["whatsapp_media"].get("bucket")
-            CloudUtility.delete_file(bucket, filename)
-            obj.delete()
+            ).delete()
             return "Deleted successfully"
```

Ruff (0.14.0)

Consider moving this statement to an `else` block

(TRY300)